pylabnet/gui/pyqt/external_gui.py

```diff
@@ -130,6 +131,12 @@ def __init__(self, app=None, gui_template=None, run=True, host=None, port=None,
         except:
             pass
 
+        # Apply stylesheet.
+        #app.setStyleSheet(qdarkstyle.load_stylesheet(qt_api='pyqt5'))
+
```

Comment:
not sure how this will affect all other GUI operation, but I guess this is to be addressed in #134

Reply:
The `setStyleSheet` function will apply the new stylesheet to all widgets, which don't have a specific style attribute set in the Designer. Have removed this line as it was commented out anyhow.